main.cc

```diff
@@ -676,7 +676,15 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
             utils::fb_utilities::set_broadcast_rpc_address(broadcast_rpc_addr);
 
             ctx.api_dir = cfg->api_ui_dir();
+            if (ctx.api_dir.length() > 0 && ctx.api_dir[ctx.api_dir.length()] != '/') {
```

Comment:
Shouldn't that be `ctx.api_dir[ctx.api_dir.length() - 1]`?

Also, can use sstring::empty() and sstring::back() for simplicity.

Reply:
You're right about the two